core/android-transitiveDependencies

```diff
@@ -1,8 +1,12 @@
 Dependencies List
 
 com.datadoghq:dd-sdk-android-core:2.16.0                        :  734 Kb
+com.squareup.okhttp3:okhttp:4.12.0                              :  771 Kb
+com.squareup.okio:okio-jvm:3.6.0                                :  351 Kb
+org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.9.10                  :  959 b 
+org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.9.10                  :  965 b 
```

Comment:
this update is _technically_ wrong if seen what is exposed to the client side, `compileOnly` won't make it to the publication https://docs.gradle.org/current/userguide/dependency_configurations.html#sub:what-are-dependency-configurations